crates/contracts/Cargo.toml

```diff
@@ -26,4 +26,4 @@ tokio = { workspace = true }
 [build-dependencies]
 anyhow = "1"
 ethers = { workspace = true, features = ["solc-full"] }
-ethers-solc = "2.0.8"
+ethers-solc = { git = "https://github.com/gakonst/ethers-rs", rev = "fa3017715a298728d9fb341933818a5d0d84c2dc" }
```

Comment:
Could you please give more info on why 2.0.8 not working and what you are fixing? I don't understand stand the intention here

Reply:
Thank you for the review!

In my local machine, `2.0.9` introduces breaking changes and does not compile - also to keep in sync w/ the workspace version!

Reply:
Would you mind post the compile error here?